Userland/Libraries/LibGfx/Painter.cpp

```diff
@@ -486,6 +486,9 @@ void Painter::draw_ellipse_intersecting(const IntRect& rect, Color color, int th
 {
     VERIFY(scale() == 1); // FIXME: Add scaling support.
 
+    if (thickness <= 0)
```

Comment:
`thickness` being signed is a bit :acid2: ...

Reply:
Yeah, it wants to either be unsigned, or possibly just a float. But this works for now.